pkg/cli/cmd/resource/delete/delete.go

```diff
@@ -33,8 +33,9 @@ import (
 )
 
 const (
-	deleteConfirmationWithoutApplication = "Are you sure you want to delete resource '%v' of type %v from environment '%v'?"
-	deleteConfirmationWithApplication    = "Are you sure you want to delete resource '%v' of type %v in application '%v' from environment '%v'?"
+	deleteConfirmationWithoutApplicationOrEnvironment = "Are you sure you want to delete resource '%v' of type %v?"
+	deleteConfirmationWithoutApplication              = "Are you sure you want to delete resource '%v' of type %v from environment '%v'?"
+	deleteConfirmationWithApplication                 = "Are you sure you want to delete resource '%v' of type %v in application '%v' from environment '%v'?"
```

Comment:
Why are some `%v`s are in single quotes and some are not?

Reply:
We put quotes around user-provided names. Right now we don't do that for types. This is consistent with the other places in the CLI where we mention types.